pkgs/applications/misc/html5validator/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildPythonApplication, fetchFromGitHub, nose, openjdk, stdenv }:
+buildPythonApplication rec {
+  pname = "html5validator";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "svenkreiss";
+    repo = "html5validator";
+    rev = "v${version}";
+    sha256 = "130acqi0dsy3midg7hwslykzry6crr4ln6ia0f0avyywkz4bplsv";
+  };
+
+  propagatedBuildInputs = [ openjdk ];
+
+  checkInputs = [ nose ];
+  checkPhase = "PATH=$PATH:$out/bin nosetests";
+
+  meta = with stdenv.lib; {
+    description = "Command line tool that tests files for HTML5 validity";
```

Comment:
please add homepage, this can just point to the github repo

Reply:
done